src/XliffTasks/Model/XamlRuleDocument.cs

```diff
@@ -60,6 +63,22 @@ private static string GenerateIdForPropertyMetadata(XElement element)
             return $"{XmlName(grandParent)}|{AttributedName(grandParent)}|Metadata|{AttributedName(element)}";
         }
 
+        private static string? GetComment(XElement element, string attributeName)
+        {
+            foreach (var comment in element.Nodes().OfType<XComment>())
+            {
+                foreach (var line in comment.Value.Split(new[] { '\n' }, System.StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()))
```

Comment:
can the new string array allocation be moved out of the loops?

Reply:
Could move it to a field, but I don't think it'd add much value. There's another params array allocation below too, not to mention enumerators and temporary strings galore.